kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -533,17 +537,26 @@ define([
          * Scroll to the top of the job log
          */
         function doFetchFirstLogChunk() {
-            doStopLogs();
-            requestJobLog(0);
+            // check the FSM state
```

Comment:
this seems like a strange way to do things -- why would you want to stop the log updating when scrolling to the top of the logs? There are better ways to override the automatic "scroll to the bottom of the log" than this.